spring-boot-consumer/src/main/resources/application.yml

```diff
@@ -3,10 +3,15 @@ server:
   error.include-message: always
 
 solace:
-  hostUrl: "tcp://localhost:55555"
+  hostUrl: "tcp://localhost:55554"
   vpnName: "default"
   userName: "default"
   password: "default"
   reconnectionAttempts: "20"
   connectionRetriesPerHost: "5"
   topicName: "solace/samples/java/direct/pub/*"
+  topicSubscriptions:
```

Comment:
@RobertOttesen : I just have one small nitpick comment, it would be better to structure the properties better to have more control over the heirarchy in case there are multiple queues and subscriptions. 
My suggestion would be to say :
`queues:
  queue:
    name: flightInfoQ
    subscriptions: 
      - "solace/samples/java/persistent/flight/boarding/1234/>"
      - "solace/samples/java/persistent/flight/delayed/ewr/*"
      - "solace/samples/java/persistent/flight/boarding/5678/atl/slc"
`